type-c-service/src/wrapper.rs

```diff
@@ -200,7 +200,7 @@ impl<'a, const N: usize, C: Controller> ControllerWrapper<'a, N, C> {
     /// Process a power command
     /// Returns no error because this is a top-level function
     async fn process_power_command(&self, controller: &mut C, port: LocalPortId, command: RequestData) {
-        trace!("Processing power command: device{} {:#?}", port, command);
+        trace!("Processing power command: device{} {:#?}", port.0, command);
```

Comment:
`LocalPortId` derives `defmt::Format` and I don't see any circumstances where the `embedded-usb-pd/defmt` feature isn't set. What am I missing?

Reply:
you're missing the fact that `defmt` is not the only logging crate. Disable `defmt` and compile with `log`. ;)